features/step_definitions/stamp_duty.rb

```diff
@@ -95,3 +95,7 @@
   expect(@stamp_duty.info_table).to have_content(data[4])
   expect(@stamp_duty.info_table).to have_content(data[5])
 end
+
+Then("I see the effective tax rate is {string}") do |string|
+  expect(@stamp_duty.effective_tax).to have_content(string)
+end
```

Comment:
new line?

Reply:
not sure why this new line isn't appearing... it's there locally and commited